packages/fxa-settings/src/components/UnitRowRecoveryKey/index.test.tsx

```diff
@@ -47,20 +47,23 @@ describe('UnitRowRecoveryKey', () => {
   });
 
   it('can be refreshed', async () => {
-    renderWithRouter(
-      <MockedCache
-        account={{ recoveryKey: false }}
-        mocks={[mockAccountQuery({ recoveryKey: true })]}
-      >
-        <UnitRowRecoveryKey />
-      </MockedCache>
-    );
+    await act(async () => {
+      renderWithRouter(
+        <MockedCache
+          account={{ recoveryKey: false }}
+          mocks={[mockRecoveryKeyExistsQuery({ recoveryKey: true })]}
+        >
+          <UnitRowRecoveryKey />
+        </MockedCache>
+      );
+    });
     expect(
       screen.getByTestId('recovery-key-unit-row-header-value')
     ).toHaveTextContent('Not Set');
     await act(async () => {
       fireEvent.click(screen.getByTestId('recovery-key-refresh'));
     });
+    await new Promise((resolve) => setTimeout(resolve, 0));
```

Comment:
According to some doc, we need to wait a tick to get past the 'loading' state of the query.

Reply:
Alternatively, I think this could be turned into:

```tsx
waitFor(() => {
  expect(
    screen.getByTestId('recovery-key-unit-row-header-value')
  ).toHaveTextContent('Enabled');
});
```

Reply:
I don't think we have `waitFor` since we are versions behind on whichever testing-library package has that func.

Reply:
Huh. It looks like we do have `wait()`, the now-deprecated predecessor to `waitFor`

Reply:
Heh, true.  I can make that change.